de_web_plugin.cpp

```diff
@@ -411,6 +411,7 @@ static const SupportedDevice supportedDevices[] = {
     { VENDOR_NONE, "MS01", tiMacPrefix }, // Sonoff SNZB-03
     { VENDOR_NONE, "MSO1", tiMacPrefix }, // Sonoff SNZB-03
     { VENDOR_NONE, "ms01", tiMacPrefix }, // Sonoff SNZB-03
+    { VENDOR_NONE, "66666", tiMacPrefix }, // Sonoff SNZB-03
```

Comment:
I have this same model number appear for Sonoof SNZB-02 will this fix for both?

Reply:
Yes